predicators/execution_monitoring/expected_atoms_monitor.py

```diff
@@ -20,7 +19,6 @@ def get_name(cls) -> str:
     def step(self, state: State) -> bool:
         # This monitor only makes sense to use with an oracle
         # bilevel planning approach.
-        assert "oracle" in CFG.approach or "active_sampler" in CFG.approach
```

Comment:
are we actually fine taking out this assertion (I just took it out to get the experiments to run lol)? If so, we should remove the two lines of comments above it. If not, we should just add "maple_q" to the assertion